HPXMLtoOpenStudio/resources/hpxml.rb

```diff
@@ -2790,7 +2790,7 @@ class CoolingSystem < BaseElement
              :cooling_shr, :third_party_certification, :seed_id, :is_shared_system, :number_of_units_served,
              :shared_loop_watts, :shared_loop_motor_efficiency, :fan_coil_watts, :airflow_defect_ratio,
              :fan_watts_per_cfm, :fan_power_not_tested, :airflow_not_tested, :charge_defect_ratio,
-             :charge_not_tested, :cooling_airflow_cfm, :location]
+             :charge_not_tested, :cooling_airflow_cfm, :location, :is_packaged_system]
```

Comment:
@shorowit The is_packaged_system is not used in the workflow, this element is just added into EPvalidator. This makes me wonder whether we need to add it as an attribute to hpxml class, if we don't need to add it, then for what reason do we need to default it?

Reply:
Good question. You're right that it's not strictly needed. I think there's no harm in defaulting it and showing that default in in.xml for transparency, and maybe some day in the future we will use it in the workflow, but I also wouldn't object if you preferred to just remove it.

Reply:
On second thought, I will probably use this element in OS-ERI so I recommend leaving it in the HPXML class.